importtest.py

```diff
@@ -0,0 +1,44 @@
+from bpython.importcompletion import find_modules
+import os, sys, tempfile
+
+with tempfile.TemporaryDirectory() as import_test_folder:
+    try:
+        os.chdir(import_test_folder)
+    except:
+        print("Can't change the Current Working Directory")
+        sys.exit()
+
+    os.mkdir(os.path.join(os.getcwd(), "Level0")
```

Comment:
You're missing a closing paren on this line, that's what's causing the error on the next line